android/gradle.properties

```diff
@@ -1,3 +1,4 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
+org.gradle.java.home=/Applications/Android Studio.app/Contents/jre/Contents/Home
```

Comment:
0_o, why do u need it?

Reply:
cuz when I tried to do ./gredlew.singingReport to obtain she-1 key have faced a message smth like  grade needs java 11 =)

Reply:
https://stackoverflow.com/a/68594140/10597533

try this variant